client/components/jetpack/threat-item-subheader/index.tsx

```diff
@@ -66,11 +67,21 @@ const getThreatStatusMessage = ( translate, threat: Threat ) => {
 	return null;
 };
 
+const getAutoFixBadge = ( translate, threat: Threat, isFixable ) => {
+	if ( isFixable && threat.status !== 'fixed' ) {
+		return (
+			<Badge className={ classnames( 'threat-item-subheader__badge', 'is-auto-fix' ) }>
+				<small>{ translate( 'Auto fix' ) }</small>
```

Comment:
:information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **22** times:
`translate( 'Auto Fixes' )` *ES Score: 7*